.github/workflows/prettier-ci.yaml

```diff
@@ -0,0 +1,25 @@
+name: Prettier CI
+
+on:
+  pull_request:
+    paths:
```

Comment:
I don't think this `path` here does anything since `prettier` is running on the entire repo if that's the case we can remove it.